### PR TITLE
CI

### PR DESCRIPTION
**Pull request by GregoryVds**
Adding Travis CI + some house-keeping

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+language: c
+sudo: required
+dist: xenial
```
Comment (matttbe) [inline .travis.yml:3]: it is time to be bionic man!
Directly switch to the last LTS, no?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+install:
+    - sudo apt-get install -y python3 python3-pip
+    - sudo pip3 install scapy-python3
+    - git submodule init && git submodule update
+before_script:
+    - sudo sysctl -w net.ipv4.tcp_fastopen=5
+script:
+    - mkdir build && cd build && cmake .. && make && sudo make test
```
Comment (matttbe) [inline .travis.yml:11]: why do you need `sudo` for the tests? (I didn't check what was done there but I guess you don't need `sudo`)

**Review by matttbe — CHANGES_REQUESTED** [review]
Looks good!
Just two comments about the Travis tests ;-)

File: .travis.yml (hunk 1 above)
Comment (GregoryVds) [reply .travis.yml:3]: Agreed! Unfortunately Travis's default is Trusty (14), and the latest they support is Xenial (16)
So I already switched to the latest they support :-)

File: .travis.yml (hunk 2 above)
Comment (GregoryVds) [reply .travis.yml:11]: The tests need to capture packets on loopback iface (I could add the capabilities but easier enough like this)

File: .travis.yml (hunk 1 above)
Comment (matttbe) [reply .travis.yml:3]: :+1:

File: .travis.yml (hunk 2 above)
Comment (matttbe) [reply .travis.yml:11]: :+1:

**Comment by GregoryVds** [comment]
@matttbe Build is finally green :-) Can you have a last look before I merge this?

**Review by matttbe — APPROVED** [review]
thx for the updates!

**Pull request merged**